### DISCUSSION TITLE
Is there a way to get to do another list of posts similar to blog posts but with projects?

### DISCUSSION THREAD

**Discussion by JerrickLiu** [daattali/beautiful-jekyll#807] [Q&A]
I'd like a blog section and a projects section for my website. Is there a way I can another section called projects that does a list of posts similar to the ```_posts``` directory? I have my blog posts in that directory and I want to make another directory that does the listing of posts similar to that

**Comment by daattali** [comment]
Hi @JerrickLiu , just a small tip: since this question is not specific to beautiful-jekyll, you might have better luck asking on a more general jekyll forum that has more viewers!

**Comment by JerrickLiu** [comment]
Hi Dean,

Thanks for the response. I’ll go ahead and do that. I don’t suppose you
have an answer to what I’m looking for though?

Best,


Jerrick

**Reply by daattali** [reply]
No sorry

**Comment by lorenzoFabbri** [comment]
Hi. I was able to replicate your website with the Projects page. One thing that does not work is the layout of each project: if I click on one of them, the text is not centered. Did you modify something else? Thanks.
